scenariolib/event_fakeSearch.go

```diff
@@ -0,0 +1,44 @@
+// Package scenariolib handles everything need to execute a scenario and send all
```

Comment:
Wow, I changed it but my PC doesn't recognize camel case in files.. so that change seems not to be commitable by my microsoft PC...